language/fi.json

```diff
@@ -0,0 +1,271 @@
+          "label": "Hyväksymisraja",
+          "description": "Prosenttimääräinen hyväksymisraja jolla tehtävä lasketaan suoritetuksi"
+        },
+        {
+          "label": "Mastering percentage",
+          "description": "Percentage that's necessary for mastering. Setting the mastering percentage below 100 % will lower the maximum possible score accordingly. It's intended to give some leeway to students, not to \"graciously accept\" solutions that do not contain all keywords."
+        },
+        {
+          "label": "Pakota Merkkikokoriippuvainen asetus",
+          "description": "Tämä asetus pakottaa päälle tai pois päältä \"Case sensitive\" asetuksen kaikille avainsanoille. ",
+          "options": [
+            {
+              "label": "Pakota"
+            },
+            {
+              "label": "Pois päältä"
+            }
+          ]
+        },
+        {
+          "label": "Pakota kirjoitusvirheiden anteeksiantaminen",
+          "description": "Tämä asetus pakottaa päälle tai pois päältä \"Forgive mistakes\" asetuksen kaikille avainsanoille",
+          "options": [
+            {
+              "label": "Pakota"
+            },
+            {
+              "label": "Pois päältä"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "label": "Painikkeen \"Check\" teksti",
+      "default": "Tarkista"
+    },
+    {
+      "label": "Painikkeen \"Retry\" teksti",
+      "default": "Yritä uudelleen"
+    },
+    {
+      "label": "Painikkeen \"Show solution\" teksti",
+      "default": "Näytä vastaukset"
+    },
+    {
+      "label": "Palauteosion otsikko, tämä osio listaa puuttuvat tai sisältyvät avainsanat.",
+      "default": "Palaute"
+    },
+    {
+      "label": "Esimerkkiratkaisun teksti",
+      "default": "Esimerkkiratkaisu"
+    },
+    {
+      "label": "Merkkiä jäljellä",
+      "default": "Merkkiä jäljellä: @chars",
+      "description": "Teksti jäljellä oleville merkeille. Voit käyttää muuttujaa @chars merkitsemään paikkaa tekstissä johon merkkien lukumäärä merkitään."
+    },
+    {
+      "label": "Ei tarpeeksi merkkejä",
+      "default": "Sinun täytyy syöttää vähintään @chars merkkiä!",
+      "description": "Teksti joka näytetään jos essee ei sisällä tarpeksi merkkejä ja henkilö yrittää palauttaa sen. Voit käyttää muuttujaa @chars merkitsemään paikkaa tekstissä johon merkkien lukumäärä merkitään."
+    },
+    {
+      "label": "Tallennusviesti",
+      "description": "Teksti joka näytetään kun viesti on tallennettu.",
+      "default": "tallennettu"
+    },
+    {
+      "label": "Sinun tuloksesi",
+      "description": "Teksti ruudunlukijoille muuttuja @score korvataan henkilön saamalla pistemäärällä ja muuttuja @total tehtävän maksimipistemäärällä.",
+      "default": "Sait @score pistettä @total pisteestä"
+    },
+    {
+      "label": "Navigaation teksti",
+      "description": "Teksti ruudunlukijoille.",
+      "default": Navigoitiin esimerkkivastaukseen tekstialueen jälkeen."
```

Comment:
Hi @Jarvil !

Thanks for your contribution!

As you can see here, github is so nice to point you to a problem here. It can detect that this file is not in proper JSON format and marks the spot. In this case, you accidentally omitted the starting double quote. If I just merged this in, the editor of Essay would not load anymore if the language was set to Finnish.

If you're unsure, you can always use services such as https://jsonlint.com/ to check the validity of those files. It can point you to semantic problems before even creating the pull request.

I'll take care of this one, so you won't have to update your pull request.

Best,
Oliver